package.json

```diff
@@ -24,6 +24,7 @@
       "pre-commit": "lint-staged && pretty-quick --staged"
     }
   },
+  "license": "iMIT",
```

Comment:
@ericboucher Do we want this to be `iMIT` or `MIT`? we should either update `LICENSE` or this line?

Reply:
MIT